File: Leetcode/1450. Number of Students Doing Homework at a Given Time/solution.go
func busyStudent(startTime []int, endTime []int, queryTime int) int {
	students := 0
	for i := 0; i < len(startTime); i++ {
		if startTime[i] <= queryTime && endTime[i] >= queryTime {
			students += 1
		}
	}
	return students
}

File: Leetcode/1450. Number of Students Doing Homework at a Given Time/solution.rs
impl Solution {
    pub fn busy_student(start_time: Vec<i32>, end_time: Vec<i32>, query_time: i32) -> i32 {
        let mut students = 0;

        for i in 0..start_time.len() {
            if start_time[i] <= query_time && end_time[i] >= query_time {
                students += 1;
            }
        }
        return students;
    }
}
